core-build/gulp-core-build-sass/src/SassTask.ts

```diff
@@ -322,4 +274,38 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
 
     return url;
   }
+
+  private _getStylesExportString(classMap: Object): string {
```

Comment:
A more specific type should get plumbed through here too.

Reply:
Thanks. Fixed.